lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -1337,13 +1337,28 @@ function dosomething_user_get_field_northstar_id($field_value) {
   return $field_value;
 }
 
+/**
+ * Return the specified Drupal user/global user's Northstar ID.
+ *
+ * @param  string $drupal_id
+ * @return mixed
+ */
+function dosomething_user_get_northstar_id($drupal_id = NULL) {
```

Comment:
It is using `dosomething_user_get_field()`. Based on where this function is being called/used, we either have the user object, or just the user's drupal id, so this is a small helper to deal with then properly calling the `dosomething_user_get_field()` with the required params.